frontend/src/index.css

```diff
@@ -8,16 +8,20 @@
   --text-light: #545454;
   --grey: rgb(196, 196, 196);
   --grey-light: #ebebeb;
-  --glass: #d2d2d287;
+  --green-light: #51f903;
+  --green-hover: #46d802;
+  --green-dark: #2f9102ce;
+  --glass: white;
   --red: red;
+  --purple: #c511ff;
 }
 
-.border {
-  border: 1px solid var(--grey);
+.border-green {
+  border: 2px solid var(--green-dark);
 }
 
 body {
-  background-image: url(https://picsum.photos/1000);
+  background-image: url(../SkyV1.png);
```

Comment:
on mobile (or other more horizontal views) the image wraps around instead of being stretched out